content/python/concepts/math-module/terms/math-isnan/math-isnan.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Math Isnan Method'
+Description: 'Math-isnan method returns True if x is a NaN (not a number) and False if x is a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Modules'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```

Comment:
```suggestion
---
Title: 'math.isnan()'
Description: 'Returns a boolean indicating whether the given value is "Not a Number" (NaN)'
Subjects:
  - 'Computer Science'
  - 'Data Science'
  - 'Web Development'
Tags:
  - 'Methods'
  - 'Modules'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Reply:
Modifications:
1. The title should be the term itself
2. Description needs to be short, starting with a verb. (In-depth explanation can be provided in the introduction paragraph below)
3. Tags should be in an alphabetical order